ckanext/dia/harvester/dcat.py

```diff
@@ -42,10 +45,20 @@ def _clean_spatial(self, spatial):
         # Convert things like "173.0039,-42.3167,174.2099,-41.0717" to
         # Polygon using templates from CSW
         if isinstance(spatial, six.string_types):
-            xmin, ymin, xmax, ymax = spatial.split(',')
-            return self.extent_template.substitute(
-                xmin=xmin, ymin=ymin, xmax=xmax, ymax=ymax
-            ).strip()
+            spatial = spatial.strip()
+            if re.match(r'^([\d\-\.]+\,){3}[\d\-\.]+$', spatial):
+                xmin, ymin, xmax, ymax = spatial.split(',')
+
+                if xmin == xmax:
+                    raise ValueError("Spatial x coordinates are the same.")
+                if ymin == ymax:
+                    raise ValueError("Spatial x coordinates are the same.")
```

Comment:
I think you meant "Spatial y coordinates are the same."

Reply:
🤦